topics/admin/tutorials/interactive-tools/tutorial.md

```diff
@@ -893,8 +894,186 @@ Once the playbook run is complete and your Galaxy server has restarted, run the
+{: .comment}
+
+In a _high availability_ setup, multiple redundant copies of Galaxy run simultaneously behind a load balancer to minimize downtime and service interruptions.
+
+As explained in [one of the previous sections](#installing-the-interactive-tools-proxy), the Galaxy Interactive Tools Proxy redirects requests to each Interactive Tool's host and port. By default, the mapping of GxIT invocations to their corresponding host/port is kept in a SQLite database known as the _Interactive Tools Session Map_.
+
+By design, [SQLite is the wrong choice for high availability setups][sqlite_situations_where_a_client_server_rdbms_may_work_better], the showstopper being that the SQLite database file would have to be shared over a network filesystem, which are usually associated with too high latencies for RDBMS use. For this reason, Galaxy and the Interactive Tools Proxy can also store the **Session Map in a PostgreSQL database**. 
+
+[sqlite_situations_where_a_client_server_rdbms_may_work_better]: https://www.sqlite.org/whentouse.html#situations_where_a_client_server_rdbms_may_work_better
+
+> <hands-on-title>Preparing the database</hands-on-title>
+> 
+> First, you need to create a database for the Interactive Tools Proxy. 
+>
+> > <warning-title></warning-title>
+> > Do **not** use the Galaxy database for this purpose. The main Galaxy database is reserved for Galaxy's core functionality, and Interactive Tools have not yet reached this stage. Since Galaxy does not expect to find the Interactive Tools Session Map in this database, storing it there can lead to errors.
+> {: .warning }
+>
+> <br>
+>
+> 1. Access PostgresSQL **in your database server**.
+> 
+>    > <code-in-title>Bash</code-in-title>
+>    > ```bash
+>    > sudo -iu postgres psql
+>    > ```
+>    > {: data-cmd="true"}
+>    {: .code-in}
+>
+>    > <code-out-title>SQL</code-out-title>
+>    > ```
+>    > psql (10.12 (Ubuntu 10.12-0ubuntu0.18.04.1))
+>    > Type "help" for help.
+>    >
+>    > postgres=#
+>    > ```
+>    {: .code-out}
+>
+> 2. Create a `gxitproxy` database to store the Interactive Tools Session Map.
+>
+>    > <code-in-title>SQL</code-in-title>
+>    > ```sql
+>    > CREATE DATABASE gxitproxy;
+>    > ```
+>    > {: data-cmd="true"}
+>    {: .code-in}
+>
+>    > <code-out-title>SQL</code-out-title>
+>    > ```
+>    > CREATE DATABASE
+>    > ```
+>    {: .code-out}
+>
+> 3. For simplicity, the same user that operates on the Galaxy main database, typically named `galaxy`, is also going to operate on this one. Make this user the owner of the new database.
+>
+>    > <code-in-title>SQL</code-in-title>
+>    > ```sql
+>    > ALTER DATABASE gxitproxy OWNER TO galaxy;
+>    > ```
+>    > {: data-cmd="true"}
+>    {: .code-in}
+>
+>    > <code-out-title>SQL</code-out-title>
+>    > ```
+>    > ALTER DATABASE
+>    > ```
+>    {: .code-out}
+>
+> 4. Sign out of the `postgres` database using `exit`. Then connect to the `gxitproxy` database as `galaxy`.
```

Comment:
you could replace steps 1+2+3 with the single CLI command: `sudo -u postgres createdb -O galaxy gxitproxy`, which would avoid students potentially missing the "logout and log in again" steps.

Reply:
Thanks, that makes the hands-on significantly shorter too.

Addressed in 72e272f.